Makefile

```diff
@@ -1,33 +1,35 @@
 GORELEASER_VERSION := v1.17.2
 
-.PHONY: build # compile natively based on the system
+# Compile natively based on the current system
+.PHONY: build 
 build:
-ifneq "" "$(findstring NT,$(shell uname))" # build for Windows
+ifneq "" "$(findstring NT,$(shell uname))" # windows
```

Comment:
As a side note: this line actually doesn't work at all on Windows, since uname isn't a valid command (at least in current Windows 10 Home)... we should fix this at some point 😅